docs/how-tos/manage-fleet.md

```diff
@@ -25,69 +25,155 @@ For example, you might have 30 robots in one warehouse and 500 in another.
+
+{{< table >}}
+{{% tablestep link="/cloud/organizations/" %}}
+**1. Create organizations**
+
+1. Log into [Viam app](https://app.viam.com) in a web browser.
+1. Click the dropdown in the upper-right corner of the **FLEET** page and use the **+** button to create a new organization.
+   Name the organization and click **Create**.
+1. Create additional organizations as needed.
+
 {{% /tablestep %}}
+{{% tablestep link="/cloud/locations/" %}}
+**2. Create locations**
+
+1. Click **FLEET** in the upper-left corner of the page.
+   A new location called `First Location` is automatically generated for you.
+
+   Click the pencil icon next to the location name and give it a name for what it represents for your use case.
+   Then click **Save**.
+
+1. Create additional locations as needed using **New location** interface, on the left of the **FLEET** page.
+
+{{% /tablestep %}}
+{{% tablestep link="/cloud/locations/" %}}
+**3. Create sub-locations**
+
+If needed, you can add further sub-locations to, for example, differentiate groups of machines within an office.
+
+To add a sub-location:
+
+1. Add a new location using the same **New location** interface.
+
+1. At the bottom of the location’s page, use the New Parent Location dropdown to choose a parent location.
+   Click **Change**.
+
+   {{<imgproc class="aligncenter" src="/tutorials/air-quality-fleet/locations-done.png" resize="x900" declaredimensions=true alt="The New York Office fleet page. The left Locations navigation panel lists Antonia's Home and RobotsRUs, with New York Office and Oregon Office nested inside RobotsRUs." >}}
+
+You can nest locations up to three levels deep.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Manage Access with Role-Based Access Control
+
+To collaborate with others on your machines, you need to invite them to your organizations, locations, or machines:
+
+{{< table >}}
 {{% tablestep link="/cloud/rbac/" %}}
-**3. Invite other users and assign permissions**
+**1. Invite users and assign permissions**
 
-Invite other users to an organization or a location to [share access](/cloud/#use-viam-for-collaboration) to the machines within it.
+If you have the **Owner** role, you can invite new users to an organization or a location to [share access](/cloud/#use-viam-for-collaboration) to the machines within it.
 Assign each user a role (owner or operator) to manage permissions.
 
 {{<imgproc src="/fleet/app-usage/limit-access.png" resize="1000x" style="max-width: 600px" class="aligncenter" declaredimensions=true alt="Limit user access">}}
 
+Users with owner access to a location or organization, can collaborate on the [machines](/cloud/machines/) within it.
+
 {{% /tablestep %}}
 {{% tablestep link="/installation/" %}}
-**4. Connect machines to the cloud**
+**2. Use API keys**
+
+You and anyone with owner access, can also create API keys for programmatic access to machines, locations, or organizations.
```

Comment:
```suggestion
You (and anyone with owner access) can create API keys for programmatic access to machines, locations, or organizations.
```
[opt] I think this reads better? And I'm not sure that "also" makes sense/is necessary here